jekyll/_plugins/uid-permalinks.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+# Error raised when two page UIDs collide, meaning two different pages will
+# have the same URL.
+class PageUIDCollisionException < StandardError
+  def initialize(msg="Page UID hashes collided.")
```

Comment:
will someone who authors documentation see this message in CI? Should we provide more context to help them understand what the right next step is once you see this message? Some sort of message explaining how it can be fixed?

Reply:
Added a more descriptive string to the message on the Error.  Hopefully this will make debugging a failed CI easier!